pkg/cmd/codespace/delete.go

```diff
@@ -187,6 +190,17 @@ func (a *App) Delete(ctx context.Context, opts deleteOptions) (err error) {
 		}
 		return nil
 	})
+
+	if a.io.IsStdoutTTY() && deletedCodespaces > 0 {
```

Comment:
I *think* that the message that some codespaces failed to delete is already printed out, because on line 189 we'll return an error saying as much. I wonder if we should have that error specify the exact number so that the messaging is cleaner. Either way, it looks like if there is an error we would currently write `Successfully deleted <n> codespaces` to `stdout` and `some codespaces failed to delete` to `stderr` which is a bit redundant

Reply:
@cmbrose yes, but I find it confusing if the message on `stdout` makes it sound like everything was successful. I'd rather it print a warning in this case since there was some success and some failure. The exact details of the failures are written to `stderr` already.

Reply:
@jkeech yeah I wasn't disagreeing with you, just saying that whatever the messaging ends up being we should look at what the entire output looks like and see if it makes sense. I think that if there's an error, the output would look something like this (assume 1 codespace successfully deletes and one fails):

```
error deleting codespace "abcd": an error message
1 codespaces deleted successfully, but 1 codespaces failed to delete
some codespaces failed to delete
```

There's a lot of redundancy there, it seems like it would be a good time to clean it up.

Reply:
True, wondering if we should not print to stdout only in case the command is successful else in case of error or partial success print to stderr? Maybe replace `some codespaces failed to delete` with `successfully deleted 1 codespace, but failed to delete 1`

Reply:
It's going to be hard to get totally rid of the final `some codespaces failed to delete` message because that is coming from `return err` which is something that we don't want to remove. Given that, make we use separate messages to specify success and delete count - some example cases:

All deletes succeed:
```
<n> codespace(s) deleted successfully
```

Mixed success:
```
error deleting codespace "abcd": an error message
<n> codespace(s) deleted successfully
<m> codespace(s) failed to delete
```

All deletes fail:
```
error deleting codespace "abcd": an error message
<m> codespace(s) failed to delete
```

We should be able to do that pretty easily with the current code - just update the error message on 189 and keep this success message as currently written

Reply:
yes this will bee fairly easy, will make the change.

Reply:
Made this change